README.md

```diff
@@ -126,39 +131,27 @@ By design Services should never expose DB Entities outside. All communication wi
 [Mapstruct](https://mapstruct.org/) is advised to use for mapping Entities to DTOs and vice versa.
 
 Typical Service & DTO pattern:
-```java
-    @Override
-    public ExampleEntityFirstDto save(ExampleEntityFirstDto exampleEntityFirstDto) {
-        ExampleEntityFirst exampleEntityFirst = exampleEntityFirstMapper.toEntity(exampleEntityFirstDto);
-        exampleEntityFirst = exampleEntityFirstRepository.save(exampleEntityFirst);
-        return exampleEntityFirstMapper.toDto(exampleEntityFirst);
-    }
-```
-
-### CI/CD
-
-[//]: # (TODO: define travice/gitlab file)
-
-### Actuator
-
-#### Healthcheck
-
-[//]: # (TODO: need to find helathckeck for Kafka)
 
-#### Metrics
-Metrics should be collected in Prometheus format using Micrometer.
```

Comment:
I think it worth adding how to deal with those metrics

Reply:
done